GitUI/ScriptsEngine/IScriptsRunner.cs

```diff
@@ -8,7 +8,7 @@ public interface IScriptsRunner
         bool RunEventScripts<THostForm>(ScriptEvent scriptEvent, THostForm form)
             where THostForm : IGitModuleForm, IWin32Window;
 
-        CommandStatus RunScript<THostForm>(int scriptId, THostForm form, RevisionGridControl? revisionGrid = null)
+        CommandStatus RunScript<THostForm>(int scriptId, THostForm form, RevisionGridControl? revisionGrid = null, IRunScript.IFileListSource files = null)
```

Comment:
The type and the parameter name aren't cohesive.

Reply:
Fixed